executor/runtime/docker/docker.go

```diff
@@ -1678,7 +1679,16 @@ func (r *DockerRuntime) setupPostStartLogDirTiniHandleConnection(parentCtx conte
 func (r *DockerRuntime) setupPostStartLogDirTiniHandleConnection2(parentCtx context.Context, c *runtimeTypes.Container, cred ucred, rootFile *os.File) error { // nolint: gocyclo
 	group, errGroupCtx := errgroup.WithContext(parentCtx)
 
+	// This required (write) access to c.RegisterRuntimeCleanup
+	if err := mountContainerProcPid1InTitusInits(parentCtx, c, cred); err != nil {
```

Comment:
It was only the `/var/lib/titus-inits` path that was a problem, not `/var/lib/titus-environments`?

Reply:
Correct.